libs/libc/machine/arm/armv7-m/Make.defs

```diff
@@ -38,6 +38,10 @@ ifeq ($(CONFIG_ARMV7M_STRCMP),y)
 ASRCS += arch_strcmp.S
 endif
 
+ifeq ($(CONFIG_ARMV7M_STRCPY),y)
+ASRCS += arch_strcpy.S
```

Comment:
should we support avmv8-m

Reply:
Yes, we should. However, it can be done later (There is no rush need for it.)